zeppelin-server/src/main/resources/shiro.ini

```diff
@@ -18,14 +18,19 @@
 [users]
 # List of users with their password allowed to access Zeppelin.
 # To use a different strategy (LDAP / Database / ...) check the shiro doc at http://shiro.apache.org/configuration.html#Configuration-INISections
-admin = password
+admin = password, admin
+user1 = user1, role1
```

Comment:
@prabhjyotsingh It's not affected actually because the default option for security is none. It would be affected if `/** = authc` is activated but there's no test for it.